packages/react-ds/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FunctionComponent } from 'react'
+
+import classNames from 'classnames/bind'
+import uniqid from 'uniqid'
+
+import { IInput } from '@venice/core/models/input.model'
+import styles from '@venice/styles/components/Input.module.scss'
+
+const Input: FunctionComponent<IInput> = ({
+  id,
+  label,
+  style,
+  className,
+  error,
+  ...props
+}: IInput) => {
+  const selfId = id || uniqid(`input__`)
+  const styleContainer = classNames(
+    styles.container,
+    { [styles.invalid]: Boolean(error) },
+    className
+  )
+
+  return (
+    <div className={styleContainer} style={style}>
+      {label && (
+        <label htmlFor={selfId} className={styles.label}>
+          {label}
+        </label>
+      )}
+      <input className={styles.input} id={selfId} {...props} />
```

Comment:
@carloshrabelo Question, this `input` allows other `type`, for example `password` and `search`?

We need to explore it here.